ossec_security/datadog_checks/ossec_security/__about__.py

```diff
@@ -0,0 +1,4 @@
+# (C) Datadog, Inc. 2024-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+__version__ = '1.0.0'
```

Comment:
Do these Crest integrations follow a different release process or are they released like our other integrations in `integrations-core`. If it's the latter, let's change this to `0.0.1` since we do have a process we follow. 🙇

Reply:
Just for posterity: this version should indeed be `0.0.1` and @manan-crest please make sure that your next integration is created with `ddev` version 9.0.0+

Reply:
Updated version to 0.0.1